airbyte-integrations/connectors/source-surveymonkey/source_surveymonkey/spec.json

```diff
@@ -4,21 +4,34 @@
     "$schema": "http://json-schema.org/draft-07/schema#",
     "title": "SurveyMonkey Spec",
     "type": "object",
-    "required": ["start_date"],
+    "required": [
+      "start_date"
+    ],
     "additionalProperties": true,
     "properties": {
       "start_date": {
         "title": "Start Date",
         "type": "string",
         "description": "The date from which you'd like to replicate the data",
         "pattern": "^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}Z?$",
-        "examples": ["2021-01-01T00:00:00Z"]
+        "examples": [
+          "2021-01-01T00:00:00Z"
+        ]
       },
       "access_token": {
         "title": "Access Token",
         "type": "string",
         "airbyte_secret": true,
         "description": "API Token. See the <a href=\"https://docs.airbyte.io/integrations/sources/surveymonkey\">docs</a> for information on how to generate this key."
+      },
+      "survey_ids": {
+        "type": "array",
+        "items": {
+          "type": "string",
+          "pattern": "^[0-9]{8,9}$"
```

Comment:
Can you please provide example of  one such `survey_id` for your account?

Reply:
`55882657`, `285871905`